modules/nw-metallb-example-addresspool.adoc

```diff
@@ -62,3 +62,40 @@ spec:
   - 10.0.100.0/28
   - 2002:2:2::1-2002:2:2::100
 ----
+
+== Example: Assign address pools to services or namespaces
+You can assign IP addresses from an `IPAddressPool` to services and namespaces that you specify.
```

Comment:
After you assign these IP address, only the service or namespace can use these IP addresses, exclusively?

Reply:
yes, This reduces scope of ip address pool to specified namespace / service and MetalLB won't use this IP pool for assignment for services coming outside of this scope,